boa_engine/fuzz/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "boa_engine-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
```

Comment:
We can probably add the readme here, now that we have it.

Reply:
How do you mean?

Reply:
`readme = "README.md"`

Reply:
Ah, apparently `cargo doc` automatically picks up a `README.md` on the root of the crate:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field

Reply:
Gotcha, thanks. I don't work much with the _publishing_ side of Rust. :)